package.json

```diff
@@ -7,7 +7,7 @@
     "doc": "docs"
   },
   "scripts": {
-    "sass": "node-sass candis/app/client/styles/App.scss:candis/app/assets/css/styles.min.css --sourcemap=none --style compressed",
+    "sass": "node-sass candis/app/client/styles/App.scss candis/app/assets/css/styles.min.css --sourcemap=none --style compressed",
```

Comment:
without introducing this `space`, I could not run `yarn run node-sass`, with error of `no output directory specified`. Maybe that's why, styles.min.css is not minified anymore.

Reply:
using `sass` instead of `node-sass` in the script, will minify the css. PS: #13 introduced the use of node-sass.